fableous-fe/src/components/canvas/Canvas.tsx

```diff
@@ -934,7 +934,7 @@ const Canvas = forwardRef<ImperativeCanvasRef, CanvasProps>(
           style={{
             position: "absolute",
             width: offsetWidth,
-            borderRadius: "30px",
+            borderRadius: "24px",
```

Comment:
if intentional also change in CursorScreen, and div with classname `bg-white` in hub and controller page

Reply:
got it